Sources/TuistLoader/Models+ManifestMappers/Headers+ManifestMapper.swift

```diff
@@ -12,25 +12,36 @@ extension TuistGraph.Headers {
     ///   - manifest: Manifest representation of Headers.
     ///   - generatorPaths: Generator paths.
     static func from(manifest: ProjectDescription.Headers, generatorPaths: GeneratorPaths) throws -> TuistGraph.Headers {
-        let unfoldGlob: (AbsolutePath) -> [AbsolutePath] = { path in
+        func resolveExcluding(_ excluding: [Path]?) throws -> Set<AbsolutePath> {
+            guard let excluding = excluding else { return [] }
+            var result: Set<AbsolutePath> = []
+            try excluding.forEach { path in
+                let resolved = try generatorPaths.resolve(path: path).pathString
+                let absolute = AbsolutePath(resolved)
+                let globs = AbsolutePath(absolute.dirname).glob(absolute.basename)
+                result.formUnion(globs)
+            }
+            return result
+        }
```

Comment:
Shouldn't the excluding be relative to the glob or I remember incorrectly?

Reply:
about public init - it was placed because this init from the code before changes, I want to be sure that all current users will update on the newer version without migration of the project.swift files

Reply:
About excluding - I  just copy the logic from the resource file glob